travel_platform/api/foursquare/izi_travel_data_tests.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
```

Comment:
Well, where is your `TODO` or `FIXME` here? Or how should I know about that?

Also it should be named as it is from the beginning.